bot/facebook/api_version.go

```
@@ -0,0 +1,61 @@
+)
+
+var RequireBatchAPIVersion = MustPareGraphAPIVersion("v26.0")
+
+type APIVersion struct {
+	Major int
+	Minor int
+}
+
+func ParseAPIVersion(s string) (APIVersion, error) {
+	s = strings.TrimPrefix(s, "v")
+	var v APIVersion
```

golangci-lint (2.12.2)

[error] missing whitespace above this line (never cuddle decl)

(wsl_v5)